src-vue/src/views/SettingsView.vue

```diff
@@ -40,25 +41,30 @@
                     <language-selector/>
                 </div>
 
-                <h3>{{ $t('settings.repair.title') }}</h3>
-                <el-button type="primary" @click="openRepairWindow">
-                    {{ $t('settings.repair.open_window') }}
-                </el-button>
+                <!-- Repair window -->
+                <div class="fc_parameter__panel">
+                    <h3>{{ $t('settings.repair.title') }}</h3>
+                    <el-button type="primary" @click="openRepairWindow">
+                        {{ $t('settings.repair.open_window') }}
+                    </el-button>
+                </div>
 
-                <h3>{{ $t('settings.about') }}</h3>
+                <!-- About section -->
+                <div class="fc_parameter__panel">
+                    <h3>{{ $t('settings.about') }}</h3>
+                    <div class="fc_northstar__version" @click="activateDeveloperMode">
+                        {{ $t('settings.flightcore_version') }} {{ flightcoreVersion === '' ? 'Unknown version' : `${flightcoreVersion}` }}
+                    </div>
+                </div>
 
-                <div class="fc_northstar__version" @click="activateDeveloperMode">
-                    {{ $t('settings.flightcore_version') }} {{ flightcoreVersion === '' ? 'Unknown version' : `${flightcoreVersion}` }}
+                <!-- Testing section -->
+                <div class="fc_parameter__panel">
+                    <h3>{{ $t('settings.testing') }}</h3>
+                    <span>
+                        {{ $t('settings.enable_test_channels') }}
+                        <el-switch v-model="enableReleasesSwitch"></el-switch>
+                    </span>
                 </div>
-                <br />
-                <br />
-                UI design inspired by <el-link :underline="false" target="_blank" href="https://github.com/TFORevive/tforevive_launcher/" type="primary">TFORevive Launcher</el-link> (not yet public)
```

Comment:
You finally wanna get rid of it? haha ^^

Reply:
Yeah, since this is basically a lie 🤡